### PR TITLE
minor : Updated SuperFinalize doc

### PR DESCRIPTION
**Pull request by harshitmalik03**
Issue : #7662 
minor : Updated SuperFinalize doc

### CODE REVIEW

**Comment by romani** [comment]
There should be update in xdoc file and javadoc of Check class.
https://github.com/checkstyle/checkstyle/blob/master/src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/SuperFinalizeCheck.java

**Comment by romani** [comment]
@harshitmalik03 , do you have problems to finish this PR ?

File: src/main/resources/com/puppycrawl/tools/checkstyle/meta/checks/coding/SuperFinalizeCheck.xml [hunk 1]
```diff
@@ -15,6 +15,29 @@
  &lt;a href="https://javarevisited.blogspot.com/2012/03/finalize-method-in-java-tutorial.html"&gt;
  10 points on finalize method in Java&lt;/a&gt;.
  &lt;/p&gt;</description>
+          <properties>
+            <property default-value="^SuperFinalize.+$"
+                      name="format"
```
Comment (romani) [inline src/main/resources/com/puppycrawl/tools/checkstyle/meta/checks/coding/SuperFinalizeCheck.xml:20]: I am not sure how you get all that properties, This Check has no properties at all.

**Review by romani — CHANGES_REQUESTED** [review]
https://checkstyle.org/config_coding.html#SuperFinalize

**Comment by harshitmalik03** [comment]
> @harshitmalik03 , do you have problems to finish this PR ?

Actually I do have a problem .... I did change the xdoc file but that too by just looking at the example you gave and i don't know if that's how its done .... I still need someone to tell me more about how exactly do i do this.

**Comment by romani** [comment]
please review commits from https://github.com/checkstyle/checkstyle/issues?q=is%3Aissue+label%3A%22gsoc+first+issue%22+is%3Aclosed
it will give you idea what and how to change a code.

**Comment by harshitmalik03** [comment]
Hey,
I checked that out..... Im not getting those "List&lt;Integer&gt". I dont know what these are and what are these for?

**Comment by romani** [comment]
@harshitmalik03 , please create PR when you pass `mvn verify` on local and you will see your examples in website.
Here is how to generate website on local - https://github.com/checkstyle/checkstyle/wiki/How-to-run-certain-phases-and-validations#how-to-generate-website-only

**Pull request closed without merging**